mkosi/resources/mkosi.md

```diff
@@ -1875,6 +1875,22 @@ use `%%`. The following specifiers are understood:
 | `ImageVersion=`    | `%v`      |
 | `Profile=`         | `%p`      |
 
+There are also specifiers that are independent of settings:
+
+| Specifier | Value                                   |
+|-----------------------------------------------------|
+| `%C`      | Parent directory of current config file |
```

Comment:
```suggestion
| Specifier | Value                                   |
|-----------|-----------------------------------------|
| `%C`      | Parent directory of current config file |
```
Otherwise the table rendering is broken